### ISSUE TITLE
why my accuracy don't increase?

### ISSUE THREAD

**Issue by henryccl** [rcouturier/steganalysis_with_CNN_and_SRM#2]
![Screenshot from 2019-06-12 10-52-10](https://user-images.githubusercontent.com/44051761/59320505-2c589f80-8d01-11e9-942c-2de3250468d4.png)

**Comment by rcouturier** [comment]
Hello,
There are several reasons:
Your payload is maybe too small
You must use more epoch, in many cases we used 1000 epochs
Can you give me more details on the experiment you used?

Raphaël

**Comment by henryccl** [comment]
I use your code in conv_stego20.py.I  change nothing except the
 cover and stego address  in fact this is the first time for me to train cnn
for steganalysis ,I use 2000 epoch and payload in 0.4 for s-uniward ,but I 
didn't trian it over.Cause  in the begining of several epochs I find the 
accuracy is always approach 0.5  that makes me feel wrong, I thought  maybe 
something was mistaken(my dataset?) Is that  accuracy normal?  If is 
how many epochs should I train so that I can get a accuracy which 
beyond 0.6?how long did you take for 2000 epochs and payload in 0.4?
![2D@_P1TW6H3{_GG37QFSE29](https://user-images.githubusercontent.com/44051761/59351197-6c933e80-8d50-11e9-8bce-69518436f126.png)

**Comment by rcouturier** [comment]
Yes it is normal at the first epochs that the accuracy is very low (as on your screenshot)
For example, in 2016 with a GPU K40, it took between 3 and 7 days to obtain good results.
So you need to be patient. Do you have a GPU on your machine? I hope so, otherwise you will need to wait for a very long time.

**Comment by henryccl** [comment]
oh，thank you really really much！My gpu isn't that good, can you give me your weights which has been pre-trained?

**Comment by henryccl** [comment]
oh，thank you very very much！My gpu isn't that good, can you give me your weights which has been pre-trained?

**Comment by rcouturier** [comment]
In fact, as this work is quite old, we didn't keep the weights, sorry

**Comment by henryccl** [comment]
That‘s all right ,thank you all the same
